Block/Callback/Ordersummary.php

```diff
@@ -63,26 +72,26 @@ class Ordersummary extends \Magento\Framework\View\Element\Template
      * OrderSummary constructor.
      *
      * @param \Magento\Framework\View\Element\Template\Context $context
-     * @param \Magento\Sales\Model\OrderFactory                $orderFactory
-     * @param \Magento\Framework\App\Request\Http              $request
-     * @param \Magento\Sales\Model\Order\Config                $orderConfig
+     * @param OrderFactory                                     $orderFactory
+     * @param Http                                             $request
+     * @param Config                                           $orderConfig
      * @param \Magento\Framework\App\Http\Context              $httpContext
-     * @param \Magento\Sales\Api\OrderRepositoryInterface      $orderRepository
+     * @param OrderRepositoryInterface                         $orderRepository
      * @param \Magento\Framework\App\Http\Context              $renderer
-     * @param \Magento\Catalog\Model\ProductRepository         $productRepository
-     * @param \Magento\Framework\Pricing\Helper\Data
+     * @param ProductRepository                                $productRepository
+     * @param Data
      * @param array                                            $data
      */
     public function __construct(
         \Magento\Framework\View\Element\Template\Context $context,
```

Comment:
Why not also import this class, you can avoid the conflict with others with the same name using an alias:
```php
             \Magento\Framework\View\Element\Template\Context as TemplateContext;
```
```suggestion
         TemplateContext $context, 
```

Reply:
Done